### PR TITLE
#5 doc: add prerequisites

### LINKED ISSUE #5: npm run build fails: sponge missing
Hi there,
According to this [line](https://github.com/derhuerst/berlin-gtfs-rt-server/blob/18a61a38f0031e6dc701ab36e176323b07b273d2/readme.md?plain=1#L35) I called this instruction

```shell
npm run build
```

, that came back with this reply.

```
npm run build

> berlin-gtfs-rt-server@4.0.0 build
> ./build.sh && npm run docs

+ grep '^PG'
PGPORT=<secret>
PGPASSWORD=<secret>
PGUSER=<secret>
PGDATABASE=<secret>
PGHOST=<secret>
+ NODE_ENV=production
+ node_modules/.bin/gtfs-to-sql -d --trips-without-shape-id --routes-without-agency-id -- gtfs/agency.csv gtfs/calendar.csv gtfs/calendar_dates.csv gtfs/frequencies.csv gtfs/routes.csv gtfs/stop_times.csv gtfs/stops.csv gtfs/transfers.csv gtfs/trips.csv
+ psql -b
+ sponge
./build.sh: line 24: sponge: command not found
PostGraphile PostgreSQL user's password: <secret>
PostrREST PostgreSQL user's password: <secret>
```

I am wondering about this

```
./build.sh: line 24: sponge: command not found
```

and this

```
PostGraphile PostgreSQL user's password: <secret>
PostrREST PostgreSQL user's password: <secret>
```

snippet. Any hint why `sponge` is not found and the GTFS feed is not imported into PostgreSQL?

Cheers!

### PR DESCRIPTION
**Pull request by dancesWithCycles**
Hi there,
According to the discussion in issue #5, I added those prerequisites.

I guessed the requirement for the PostGIS extension from reverse engineering. Is that right? Are more PostgreSQL extensions necessary that I missed?

Cheers!

---

edit @derhuerst: fixes #5

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,10 +17,12 @@ This project uses [`hafas-client`](https://github.com/public-transport/hafas-cli
 
 `berlin-gtfs-rt-server` needs access to a [Redis](https://redis.io/) server, you can configure a custom host/port by setting the `REDIS_URL` environment variable.
 
-It also needs access to a [PostgreSQL](https://www.postgresql.org) 14+ server; Pass custom [`PG*` environment variables](https://www.postgresql.org/docs/14/libpq-envars.html) if you run PostgreSQL in an unusual configuration.
+It also needs access to a [PostgreSQL](https://www.postgresql.org) 14+ server with the [PostGIS](https://postgis.net/) extension activated; Pass custom [`PG*` environment variables](https://www.postgresql.org/docs/14/libpq-envars.html) if you run PostgreSQL in an unusual configuration.
```
Comment (derhuerst) [inline readme.md]: I'm trying to make the distinction between installing an extension into the RDBMS and activating/loading it into a DB more clear. Unfortunately, not even the [`CREATE EXTENSION` docs](https://www.postgresql.org/docs/14/sql-createextension.html) are consistent about this.

```suggestion
It also needs access to a [PostgreSQL](https://www.postgresql.org) 14+ server with [PostGIS](https://postgis.net/) installed; Pass custom [`PG*` environment variables](https://www.postgresql.org/docs/14/libpq-envars.html) if you run PostgreSQL in an unusual configuration.
```

File: readme.md [hunk 2]
```diff
@@ -17,10 +17,12 @@ This project uses [`hafas-client`](https://github.com/public-transport/hafas-cli
...
 
 It also needs access to a [NATS Streaming](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan) server (just follow its [installation guide](https://nats-io.gitbook.io/legacy-nats-docs/nats-streaming-server-aka-stan/stan-nats-streaming-server/installing)); Set the `NATS_STREAMING_URL` environment variable if you run it in an unusual configuration.
 
+It also needs the command line instruction [Sponge](https://linux.die.net/man/1/sponge) from the [Moreutils](https://repology.org/project/moreutils/information) package installed on the host computer.
```
Comment (derhuerst) [inline readme.md]:
```suggestion
It also needs the [`sponge` command](https://linux.die.net/man/1/sponge) from the [moreutils package](https://repology.org/project/moreutils/information) to be installed.
```

**Comment by derhuerst** [comment]
Thanks! I have amended, force-pushed & merge your commits as cffcc7d.

**Comment by derhuerst** [comment]
Do you want to be mentioned as a contributor [in `package.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)? If yes, how? `Begerad, Stefan <dialog@swingbe.de>`?

**Comment by dancesWithCycles** [comment]
> Do you want to be mentioned as a contributor [in `package.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors)? If yes, how? `Begerad, Stefan <dialog@swingbe.de>`?

Perfect! Thx!

**Pull request merged**